catalog/UMD/UMD_hansen_global_forest_change.libsonnet

```diff
@@ -9,4 +9,5 @@
   'v1.7': 'UMD/hansen/global_forest_change_2019_v1_7',
   'v1.8': 'UMD/hansen/global_forest_change_2020_v1_8',
   'v1.9': 'UMD/hansen/global_forest_change_2021_v1_9',
+  'v1.10': 'UMD/hansen/global_forest_change_2022_v1_10',
```

Comment:
Please rebase from google main. The versions.libsonnet now takes an array of objects and should look like this:

```
[
  {'v1.0': 'UMD/hansen/global_forest_change_2013'},
  {'v1.1': 'UMD/hansen/global_forest_change_2014'},
  {'v1.2': 'UMD/hansen/global_forest_change_2015'},
  {'v1.3': 'UMD/hansen/global_forest_change_2015_v1_3'},
  {'v1.4': 'UMD/hansen/global_forest_change_2016_v1_4'},
  {'v1.5': 'UMD/hansen/global_forest_change_2017_v1_5'},
  {'v1.6': 'UMD/hansen/global_forest_change_2018_v1_6'},
  {'v1.7': 'UMD/hansen/global_forest_change_2019_v1_7'},
  {'v1.8': 'UMD/hansen/global_forest_change_2020_v1_8'},
  {'v1.9': 'UMD/hansen/global_forest_change_2021_v1_9'},
  {'v1.10': 'UMD/hansen/global_forest_change_2022_v1_10'},
]
```